src/libraries/Common/tests/System/Net/Http/ResponseStreamTest.cs

```diff
@@ -230,6 +230,99 @@ await client.GetAsync(remoteServer.EchoUri, HttpCompletionOption.ResponseHeaders
 
 #if NET
 
+        public static IEnumerable<object[]> HttpMethods => new object[][]
+        {
+            new [] { HttpMethod.Get },
+            new [] { HttpMethod.Head },
+            new [] { HttpMethod.Post },
+            new [] { HttpMethod.Put },
+            new [] { HttpMethod.Delete },
+            new [] { HttpMethod.Options },
+            new [] { HttpMethod.Patch },
+        };
+
+        public static IEnumerable<object[]> HttpMethodsAndAbort => new object[][]
+        {
+            new object[] { HttpMethod.Get, "abortBeforeHeaders" },
+            new object[] { HttpMethod.Head , "abortBeforeHeaders"},
+            new object[] { HttpMethod.Post , "abortBeforeHeaders"},
+            new object[] { HttpMethod.Put , "abortBeforeHeaders"},
+            new object[] { HttpMethod.Delete , "abortBeforeHeaders"},
+            new object[] { HttpMethod.Options , "abortBeforeHeaders"},
+            new object[] { HttpMethod.Patch , "abortBeforeHeaders"},
+
+            new object[] { HttpMethod.Get, "abortAfterHeaders" },
+            new object[] { HttpMethod.Post , "abortAfterHeaders"},
+            new object[] { HttpMethod.Put , "abortAfterHeaders"},
+            new object[] { HttpMethod.Delete , "abortAfterHeaders"},
+            new object[] { HttpMethod.Options , "abortAfterHeaders"},
+            new object[] { HttpMethod.Patch , "abortAfterHeaders"},
+
+            new object[] { HttpMethod.Get, "abortDuringBody" },
+            new object[] { HttpMethod.Post , "abortDuringBody"},
+            new object[] { HttpMethod.Put , "abortDuringBody"},
+            new object[] { HttpMethod.Delete , "abortDuringBody"},
+            new object[] { HttpMethod.Options , "abortDuringBody"},
+            new object[] { HttpMethod.Patch , "abortDuringBody"},
+
+       };
+
+        [MemberData(nameof(HttpMethods))]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser))]
+        public async Task BrowserHttpHandler_StreamingResponse(HttpMethod method)
```

Comment:
won't these new tests fail until the EchoServer is updated with your changes? I'm not sure if you can do it in the same PR

Reply:
We are always using the local HTTP server for browser testing. The assembly is plugin into xharness.
I don't even know how to deploy to the azure instance :)

Reply:
ah interesting, thanks